### PR TITLE
Adding secondary nav to Streams page

### PR DESCRIPTION
**Pull request by manjuapu**
Added secondary navigation to Streams sub-pages:
 - quickstart.html
 - core-concepts.html
 - developer-guide.html
 - tutorial.html

### Committer Checklist (excluded from commit message)
- [x] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by manjuapu** [comment]
@guozhangwang Please review. Thanks!!

File: docs/streams/core-concepts.html [hunk 1]
```diff
@@ -19,7 +19,17 @@
...
 <script id="content-template" type="text/x-handlebars-template">
     <h1>Core Concepts</h1>
-
+    <div class="sub-nav-sticky">
+      <div class="sticky-top">
+        <div style="height:35px">
+          <a href="/{{version}}/documentation/streams/">Introduction</a>
+          <a href="/{{version}}/documentation/streams/developer-guide">Developer Guide</a>
+          <a href="/{{version}}/documentation/streams/core-concepts">Concepts</a>
+          <a class="active-menu-item" href="/{{version}}/documentation/streams/quickstart">Run Demo App</a>
```
Comment (joel-hamill) [inline docs/streams/core-concepts.html]: shouldn't `class="active-menu-item"` be on Concepts?

File: docs/streams/developer-guide.html [hunk 2]
```diff
@@ -19,7 +19,17 @@
...
 <script id="content-template" type="text/x-handlebars-template">
     <h1>Developer Guide for Kafka Streams API</h1>
-    
+    <div class="sub-nav-sticky">
+      <div class="sticky-top">
+        <div style="height:35px">
+          <a href="/{{version}}/documentation/streams/">Introduction</a>
+          <a href="/{{version}}/documentation/streams/developer-guide">Developer Guide</a>
+          <a href="/{{version}}/documentation/streams/core-concepts">Concepts</a>
+          <a class="active-menu-item" href="/{{version}}/documentation/streams/quickstart">Run Demo App</a>
```
Comment (joel-hamill) [inline docs/streams/developer-guide.html]: shouldn't `class="active-menu-item"` be on Developer Guide?

File: docs/streams/tutorial.html [hunk 3]
```diff
@@ -18,7 +18,17 @@
...
 <script id="content-template" type="text/x-handlebars-template">
     <h1>Tutorial: Write a Streams Application</h1>
-
+    <div class="sub-nav-sticky">
+      <div class="sticky-top">
+        <div style="height:35px">
+          <a href="/{{version}}/documentation/streams/">Introduction</a>
+          <a href="/{{version}}/documentation/streams/developer-guide">Developer Guide</a>
+          <a href="/{{version}}/documentation/streams/core-concepts">Concepts</a>
+          <a class="active-menu-item" href="/{{version}}/documentation/streams/quickstart">Run Demo App</a>
```
Comment (joel-hamill) [inline docs/streams/tutorial.html]: shouldn't `class="active-menu-item"` be on Tutorial: Write App?

**Review by joel-hamill — CHANGES_REQUESTED** [review]
What about upgrade-guide.html and architecture.html? Shouldn't these streams topics also be part of the header?

**Comment by guozhangwang** [comment]
@manjuapu could you update the PR title / description (it was indicated in the template) and mark the checklist if you have done so.

**Comment by manjuapu** [comment]
@joel-hamill  upgrade-guide.html and architecture.html is not included in the design, so I have not added it. You can see it in this page - https://kafka.apache.org/documentation/streams/

**Comment by guozhangwang** [comment]
LGTM. Merged to trunk and 1.0.

**Comment by asfgit** [comment]
FAILURE
 7959 tests run, 5 skipped, 1 failed.
--none--

**Comment by asfgit** [comment]
SUCCESS 
 8067 tests run, 5 skipped, 0 failed.
--none--

**Comment by asfgit** [comment]
FAILURE
 7959 tests run, 5 skipped, 1 failed.
--none--

**Comment by joel-hamill** [comment]
@manjuapu not sure what you mean https://github.com/apache/kafka/pull/4222#issuecomment-344743054. why are they not included? (cc @derrickdoo )

**Pull request closed without merging**